Source/Core/DolphinQt/Host.cpp

```diff
@@ -287,6 +302,15 @@ void Host_TitleChanged()
 #endif
 }
 
+WindowInfo Host_GetRenderWindowInfo()
+{
+#ifdef _WIN32
+  return WindowInfo{Host::GetInstance()->GetRenderWindowInfo()};
```

Comment:
sorry if i missed it but why is this windows specific? can't you get usable platform-independent values out of Qt? or at least let Qt give you the platform-dependent values?

Reply:
Uhhhhhhhh, it's platform-dependent because I didn't think about that. I will look at getting QT to give those values instead of this. Thanks :>

Reply:
You still need to remove the ifdef here.

Reply:
I did on mine. Apologies, I don't know why that didn't get pushed.